builtin/game/falling.lua

```diff
@@ -5,43 +5,43 @@
 --
 
 core.register_entity(":__builtin:falling_node", {
-	initial_properties = {
-		physical = true,
-		collide_with_objects = false,
-		collisionbox = {-0.5,-0.5,-0.5, 0.5,0.5,0.5},
-		visual = "wielditem",
-		textures = {},
-		visual_size = {x=0.667, y=0.667},
-	},
+	visual = "wielditem",
+	visual_size = {x = 0.667, y = 0.667},
+	textures = {},
+	physical = true,
+	is_visible = false,
+	collide_with_objects = false,
+	collisionbox = {-0.5, -0.5, -0.5, 0.5, 0.5, 0.5},
 
 	node = {},
 
 	set_node = function(self, node)
 		self.node = node
-		local prop = {
+		self.object:set_properties({
 			is_visible = true,
-			textures = {node.name},
-		}
-		self.object:set_properties(prop)
+			textures = {node.name}
+		})
 	end,
 
 	get_staticdata = function(self)
-		return self.node.name
+		return core.serialize(self.node)
```

Comment:
It should be core.serialize because self.node has param2 value.
Also I added supporting old staticdata (line 38).